build.gradle

```
@@ -44,29 +44,46 @@ repositories {
   mavenCentral()
 }
 
-apply plugin: 'com.github.johnrengelman.shadow'
-
-shadowJar {
-  classifier = null
-}
-
 dependencies {
   testImplementation 'org.apache.logging.log4j:log4j-core:2.17.1'
 
   implementation 'io.netty:netty-all:4.1.65.Final'
   implementation group: 'commons-io', name: 'commons-io', version: '2.8.0' // used from Netty
-  implementation 'org.apache.logging.log4j:log4j-api:2.17.1'
-
+  compileOnly 'org.apache.logging.log4j:log4j-api:2.17.1' // provided by Logstash
+  compileOnly group: 'org.jruby', name: 'jruby-complete', version: "9.1.13.0" // provided by Logstash
```

No need to bump netty and bouncy castle and versions in this PR but this is a new line:

```suggestion
  compileOnly group: 'org.jruby', name: 'jruby-complete', version: "9.2.20.1" // provided by Logstash
```

---

going to leave it as is for now - there isn't a `required_ruby_version >= ...` thus plugin might still work on old LS